### PR TITLE
feat(INT-6665): review GCP permissions

### PR DESCRIPTION
**Pull request by gastonyelmini**
Review GCP permissions and generate permissions list.

### CODE REVIEW

File: husky.config.js [hunk 1]
```diff
@@ -1 +1,8 @@
-module.exports = require('@jupiterone/integration-sdk-dev-tools/config/husky');
+const huskyConfig = require('@jupiterone/integration-sdk-dev-tools/config/husky');
+
+module.exports = {
+  hooks: {
+    ...huskyConfig.hooks,
+    'pre-commit': `${huskyConfig.hooks['pre-commit']} && yarn document:permissions && git add docs/jupiterone.md`,
```
Comment (gastonyelmini) [inline husky.config.js:6]: This will run SDK commands together with the `document:permissions` one.

Comment (ndowmon) [reply husky.config.js:6]: 🔥

Comment (ndowmon) [reply husky.config.js:6]: You may want to do `yarn lint-staged` afterwards... so might be worth running the huskyConfig's pre-commit _after_ your script. Not a big deal though.

Comment (gastonyelmini) [reply husky.config.js:6]: thank you for your comments! will update that

**Review by ndowmon — APPROVED** [review]

**Pull request merged**